tests/fixtures/optimizer/canonicalize.sql

```diff
@@ -10,6 +10,9 @@ SELECT CAST(1 AS VARCHAR) AS "a" FROM "w" AS "w";
 SELECT CAST(1 + 3.2 AS DOUBLE) AS a FROM w AS w;
 SELECT 1 + 3.2 AS "a" FROM "w" AS "w";
 
+SELECT '1' + 1 AS "col";
+SELECT '1' + 1 AS "col";
```

Comment:
This was previously transformed into `SELECT CONCAT('1', 1) AS "col"` which doesn't match the semantics implemented by the aforementioned dialects.

Reply:
does '1' + '1' still equal concat?

Reply:
yes